### DISCUSSION TITLE
just enable the ant+ power sensor on the setting of QZ and then use zwift with your dongle ant+ on your pc :)

### DISCUSSION THREAD

**Discussion by xczsonn** [cagnulein/qdomyos-zwift#352] [Q&A]
Can it send all the information? (watt, cadence, resistance, etc.)
and what is ant + power sensor option? it only exist ant+ cadence, ant+ heart

just enable the ant+ power sensor on the setting of QZ and then use zwift with your dongle ant+ on your pc :)

_Originally posted by @cagnulein in https://github.com/cagnulein/qdomyos-zwift/issues/349#issuecomment-855392022_

**Comment by cagnulein** [comment]
no by ant+ only power atm.
you should use bluetooth instead.
please don't open a new bug every time :)



Il giorno dom 6 giu 2021 alle ore 14:54 xczsonn ***@***.***>
ha scritto:
